frontend/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@fluent/bundle": "^0.15.0",
-    "@fluent/langneg": "^0.3.0",
-    "@fluent/react": "^0.10.0",
-    "@fluent/syntax": "^0.15.0",
+    "@fluent/bundle": "0.15.0",
+    "@fluent/langneg": "0.3.0",
```

Comment:
`@fluent/langneg` 0.4.0 is out, too, now rewritten to TypeScript. No API changes.